server.ts

```diff
@@ -1 +1,12 @@
+import { serve } from "https://deno.land/std/http/server.ts";
+
+const server = serve({ hostname: "0.0.0.0", port: 8080 });
```

Comment:
I suggest you get port value from a variable and/or obtain it from command line arguments